chart/values.yaml

```diff
@@ -220,6 +220,31 @@ ingress:
       # the name of a pre-created Secret containing a TLS private key and certificate
       secretName: ""
 
+# Enable the creation of a Kubernetes HTTPRoute and HealthCheckPolicy
+httpRoute:
+  web:
+    # Enable web ingress resource
+    enabled: false
+    # Annotations for the web HTTPRoute
+    annotations: {}
+    # The hostnames or hosts configuration for the web HTTPRoute
+    hostnames: []
+    # The Gateway API configuration
+    gateway:
+      name: your-gateway
+      namespace: namespace-for-your-gateway
```

Comment:
You can keep them empty, and check in the template if they are provided, and fail if they are not (at least name, and maybe use chart namespace as namespace for gateway). Otherwise, users will try to use it with default values, and it will fail.

Reply:
I'll leave them both empty. My guess is that k8s admins will not have the gateway running in the same namespace as Airflow.